src/app/components/framework/roman-numerals.pipe.ts

```diff
@@ -0,0 +1,58 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'asRomanNumeral',
+})
+export class RomanNumeralsPipe implements PipeTransform {
+  transform(value: number): string {
+    return this._romanize(value);
+  }
+
+  /** This function was taken from the following blog: https://blog.stevenlevithan.com/archives/javascript-roman-numeral-converter */
+  private _romanize(num) {
+    if (!+num) {
```

Comment:
please use (!(num > 0)).
Is much more readable, and will have the same behavior afaik.